app/views/practices/_adoption_status_tooltip.html.erb

```diff
@@ -1,4 +1,4 @@
-<i class="fas fa-question-circle usa-tooltip"
```

Comment:
I don't think we are using this file anymore

Reply:
Removed it.